pyDeltaRCM/deltaRCM_tools.py

```diff
@@ -293,6 +293,10 @@ def output_data(self):
                 if self.save_velocity_grids:
                     self.save_grids('velocity', self.uw, shape[0])
 
+            if self.checkpoint:
+                _msg = 'Saving checkpoint'
```

Comment:
you need to add `self.logger.info(_msg)` to the following line. 

Consider adding another test, which looks for these records in the log file too. You can see an example of reading lines in the log file [here](https://github.com/DeltaRCM/pyDeltaRCM_WMT/blob/7270b70b7fd2fc3ab89e5a8713ed478979da30ea/tests/test_deltaRCM_tools.py#L175).